docs/04-providers/01-github.md

```
@@ -0,0 +1,37 @@
+
+### Dependabot
+
+TODO
+
+### Projects
+
+TODO
+
+### Collaboration
+
+TODO
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)